src/app/music/page.tsx

```diff
@@ -1,7 +1,49 @@
+import Link from "next/link";
+import { RxDoubleArrowDown } from "react-icons/rx";
+
 export default function Music() {
-  return(
-    <div className="flex justify-center items-center h-screen pt-10">
-      <p className="text-3xl">Coming soon!</p>
-    </div>
+  return(    
+    <>
+      <div id="songs" className="h-screen">
+        <div 
+          className="grid justify-center md:block md:justify-normal pt-10 pb-7"
+          style={{width: "85vw", margin: "auto"}}
+        >
+          <h1 className="text-5xl lg:text-6xl" style={{color: "#FFFFFF"}}>Original Songs</h1>
+        </div>
+        <hr className="block m-auto" style={{width: "85vw"}}></hr>
+        <div className="flex justify-center pt-10">
+          <iframe 
+            width="75%" 
+            height="350" 
+            frameborder="no" 
+            allow="autoplay" 
+            src="https://w.soundcloud.com/player/?url=https%3A//soundcloud.com/lotus-flower-206577421&show_playcount=false"
+          >
+          </iframe>
+        </div>
+        <div className="flex justify-center items-center h-60">
+          <Link href="#performances" className="flex flex-col items-center">
```

Comment:
A fix for later, but I will need to improve the scrolling animation to the Performances section